application/templates/sealedsecrets.yaml

```diff
@@ -12,22 +12,49 @@ metadata:
 {{- if $.Values.sealedSecret.additionalLabels }}
 {{ toYaml $.Values.sealedSecret.additionalLabels | indent 4 }}
 {{- end }}
-{{- if $.Values.sealedSecret.annotations }}
+{{- if $config.labels }}
+{{ toYaml $config.labels | indent 4 }}
+{{- end }}
+{{- if or ($.Values.sealedSecret.annotations) ($config.annotations) }}
   annotations:
+{{- if $config.annotations }}  
+{{ toYaml $config.annotations | indent 4 }}
+{{- end }}
+{{- if $.Values.sealedSecret.annotations }}  
 {{ toYaml $.Values.sealedSecret.annotations | indent 4 }}
 {{- end }}
+{{- if $config.clusterWide}}
+    sealedsecrets.bitnami.com/cluster-wide: true
+{{- end}}
+{{- end }}
 spec:
   encryptedData:
-  {{- range $key, $value := .encryptedData }}
+{{- range $key, $value := $config.encryptedData }}
     {{ $key }}: {{ $value }}
   {{- end }}
+  type: {{ $config.type | default "Opaque" }}
   template:
     metadata:
       name: {{ template "application.name" $ }}-{{ $nameSuffix }}
       namespace: {{ template "application.namespace" $ }}
-      {{- if $.Values.sealedSecret.annotations }}
+      {{- if or ($.Values.sealedSecret.annotations) ($config.annotations) ($config.clusterWide) }}
       annotations:
-     {{ toYaml $.Values.sealedSecret.annotations | indent 4 }}
+      {{- if $config.annotations }}  
+      {{ toYaml $config.annotations | indent 2 }}
+      {{- end }}
+      {{- if $.Values.sealedSecret.annotations }}  
+      {{ toYaml $.Values.sealedSecret.annotations | indent 2 }}
+      {{- end }}
+      {{- if $config.clusterWide}}
+        sealedsecrets.bitnami.com/cluster-wide: true
+      {{- end}}
+      {{- end }}
+      {{- if $config.labels }}  
```

Comment:
should have been a condition if config.labels or $.Values.sealedSecret.additionalLabels